content/applications/finance/fiscal_localizations/belgium.rst

```diff
@@ -377,70 +352,45 @@ Once the connection is established, Odoo can be synchronized with CodaBox.
-
-Error while connecting to the IAP server. Please contact Odoo support.
-  It seems that Odoo's IAP server is down or unreachable. Please contact
-  Odoo support.
-
-It seems that your CodaBox connection is not valid anymore. Please check your configuration.
-  It seems that you have either revoked the access of Odoo to your CodaBox account, or that
-  you have not completed the configuration process. In this case, you must click on the
-  :guilabel:`Revoke connection` button in the settings, and follow again the steps in
-  the :ref:`first connection <belgium/codabox-configuration-connection>` section.
-
-A previous CodaBox registration already exists. Paste the access token from a previous connection here. Odoo will generate a new one, exclusive to this database.
-  It seems that you have already your Accounting Firm with Odoo, but that the Access Token
-  is either not filled or wrong. You should follow the steps in the
-  :ref:`following connection <belgium/codabox-configuration-connection>` tab section.
-
-  If you have lost your Access Token, you have to go on the CodaBox platform,
-  and revoke the access of Odoo to your CodaBox account. Then, you will have to follow
-  the steps in the :ref:`first connection <belgium/codabox-configuration-connection>` section.
-
-The provided access token is not valid for this fiduciary. Please check your configuration. If you have lost your access token, please contact Odoo support.
-  Same as above.
-
-It seems that no connection linked to your database/VAT number exists. Please check your configuration.
-  You may obtain this error when revoking or trying to fetch files from CodaBox if
-  the pair (Accounting Firm VAT, Company VAT) is not registered yet. This may happen if
-  you have changed the company VAT after the connection was established for instance.
-  For security reasons, you have to recreate a connection for this company VAT by following the
-  steps in the :ref:`first connection <belgium/codabox-configuration-connection>` section.
-
-No files were found. Please check your configuration.
-  This may occur if the company VAT has changed after the connection was established.
-  In that case, you must click on the :guilabel:`Revoke connection` button in the settings,
-  and recreate the connection.
-
-It seems you have already created a connection to CodaBox with this fiduciary. To create a new connection, you must first revoke the old one on myCodaBox portal.
-  You must go on the CodaBox platform, and revoke the access of Odoo to your CodaBox account.
-  Then, you will have to follow the steps in the
-  :ref:`first connection <belgium/codabox-configuration-connection>` section.
+No connection exists with these accounting firms and company VAT numbers. Please check your configuration.
+  This can happen when checking the connection status, and the pair
+  (Accounting Firm VAT, Company VAT) still needs to be registered. This may happen if you have
```

Comment:
```suggestion
  This can happen when checking the connection status, and the Accounting Firm VAT and Company VAT combination still needs to be registered. This may happen if you have
```